packages/components/src/reaction_selection_menu/index.jsx

```diff
@@ -0,0 +1,101 @@
+import React, { useCallback, useRef, useState } from 'react';
+import PropTypes from 'prop-types';
+import clsx from 'clsx';
+import { invoke } from 'nebenan-helpers/lib/utils';
+import { eventCoordinates } from 'nebenan-helpers/lib/dom';
+import { useEventListener } from 'nebenan-react-hocs/lib/use_event_listener';
+import styles from './index.module.scss';
+import { useLongHover, useLongTouch } from './hooks';
+import { getHoverIndex } from './utils';
+import { REACTION_SIZE_M, REACTIONS } from '../constants';
+import ReactionIconBubble from '../reaction_icon_bubble';
+
+const ReactionSelectionMenu = ({ className, label, strings, onSelect }) => {
+  const rootRef = useRef(null);
+  const [isActive, setIsActive] = useState(false);
+  const [hoverReaction, setHoverReaction] = useState(null);
+  const itemRefs = useRef([]);
+
+  const { active: activeTouchEvents, passive: touchEvents } = useLongTouch({
+    onStart: useCallback(() => {
+      setIsActive(true);
+    }, []),
+    onEnd: useCallback(() => {
+      setHoverReaction(null);
+      setIsActive(false);
+
+      if (hoverReaction) invoke(onSelect, hoverReaction);
```

Comment:
you just set it to `null` on L24, probably need to copy it somewhere?

Reply:
Good point. This whole callback only worked because of async state updates. Will move this line up to the top